DarkUI/DarkUI/DarkUI.csproj

```diff
@@ -123,7 +123,4 @@
   <ItemGroup>
     <Content Include="Resources\grip_fill.png" />
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0" />
```

Comment:
Does the app run with this lib missing? It is used at the start of every `Program.cs` to allow the use of `Windows 1252` encoding across the apps.

Reply:
Where exactly? There is no usage of the CodePages Encoding

Reply:
AFAIK if you build and run the app, it will crash immediately with an exception saying something like "Windows 1252 encoding is not available." But that might have been a .NET 5 error which may not be present in .NET 6, but please verify.

Reply:
Script files that are not UTF-8 have their non-UTF-8 characters replaced with `?` but no exception or anything.
Any other encoding than UTF should not be supported anyway.

Reply:
The TRNG compiler (and the old TR4 one as well I think) can't compile UTF documents, they can only be Windows 1252, otherwise the compiler throws an error.

Reply:
test it and require a PR change then.

Reply:
Alright, so this was indeed a .NET 5 issue:
![image](https://github.com/user-attachments/assets/654cd035-e35a-4d76-882b-98b68d9f560f)

But in .NET 6, it seems to be fixed, sorry about that, I thought it was a general issue since moving to .NET Core, but they seem to have quietly fixed it.